rust/hyperlane-base/src/metrics/custom_metrics.rs

```diff
@@ -3,12 +3,15 @@ use std::time::Duration;
 use derive_builder::Builder;
```

Comment:
curious why custom? don't feel strongly but wondering if there's a more accurate name closer to "chain" or "agent" or something

Reply:
changed to `agent_metrics.rs` and added a file-level doc comment: `//! Metrics either related to the agents, or observed by them`